src/main/java/org/sonarlint/intellij/core/BackendService.kt

```diff
@@ -400,6 +404,20 @@ class BackendService @NonInjectable constructor(private val backend: SonarLintBa
         return initializedBackend.connectionService.validateConnection(params)
     }
 
+    fun listUserOrganizations(server: ServerConnection): CompletableFuture<ListUserOrganizationsResponse> {
```

Comment:
Very good! I have to check if we have something similar in Eclipse or if we might have the same issue but no community report as of yet!
I can check manually but we have no Eclipse-compatible project using SonarCloud if I'm not mistaken, maybe you are aware?

Reply:
It's already implemented on Eclipse side, no worries about that
For testing, you should just connect to your SC account by using GitHub for example, it should do the job